include/zephyr/bluetooth/gatt.h

```diff
@@ -199,6 +204,11 @@ typedef ssize_t (*bt_gatt_attr_read_func_t)(struct bt_conn *conn,
  *  @note The GATT server propagates the return value from this
  *  method back to the remote client.
  *
+ *  @note If this function returns ``-EINPROGRESS``, the write operation
+ *  is deferred and the application must later send the response
+ *  using @ref bt_gatt_send_write_response(). Deferred responses
+ *  are only supported when Enhanced ATT (EATT) is not active.
```

Comment:
`when Enhanced ATT (EATT) is not active.` - What does this mean specifically?

`CONFIG_BT_EATT=n`? Or just that no EATT bearers are currently connected for the specific conn?